ui_testing/testcases/header_tests/test08_usermanagement.py

```diff
@@ -238,4 +227,289 @@ def test010_update_user_role_with_save_cancel_btn(self, save):
+            self.base_selenium.LOGGER.info(
+                    ' + Assert {} (current_email) == {} (user_email)'.format(current_email, user_email))
+            self.assertEqual(current_email, user_email)
+
+    def test011_validation_user_name_email_fields(self):
+        """
+        Header: User management: Make sure when the user update name & email then press on save button,
+        red border display and he can't save
+        LIMS-6121
+        :return:
+            """
+        # from the create mode it will redirect me to the active table
+        self.base_selenium.click(element='header:user_management_button')
+        self.header_page.get_random_user()
+        self.header_page.clear_user_name()
+        self.header_page.clear_user_email()
+        self.header_page.save(save_btn='user_management:save_btn')
+        self.base_selenium.LOGGER.info('Waiting for error message')
+        validation_result = self.base_selenium.wait_element(element='general:oh_snap_msg')
+        self.base_selenium.LOGGER.info('Assert error msg')
+        self.assertEqual(validation_result, True)
+
+    def test012_delete_user_used_in_other_entity(self):
+        """
+        User management: Make sure that you can't delete any user record If this record used in other entity
+        LIMS-6407
+        :return:
+        """
+        self.base_selenium.click(element='header:user_management_button')
+        # create new user with random data
+        user_random_name = self.generate_random_string()
+        user_random_email = self.base_page.generate_random_email()
+        self.header_page.create_new_user(user_name=user_random_name,
+                                         user_email=user_random_email,
+                                         user_role='Admin', user_password='1',
+                                         user_confirm_password='1')
+
+        self.base_selenium.LOGGER.info(
+            'search to make sure that the user created '.format(user_random_name))
+        created_user = self.header_page.search(user_random_name)[0]
+        user_data = self.base_selenium.get_row_cells_dict_related_to_header(row=created_user)
+
+        self.header_page.click_on_header_button()
+        self.base_selenium.click(element='header:logout')
+
+        # use this user in any other entity so you can login with it
+        self.login_page.login(username=user_random_name, password='1')
+
+        # make sure that I can login with the user that I created it
+        time.sleep(15)
+        self.assertIn('dashboard', self.login_page.base_selenium.get_url())
+
+        self.header_page.click_on_header_button()
+        self.base_selenium.click(element='header:user_management_button')
+        self.base_page.sleep_small()
+
+        self.base_selenium.LOGGER.info(
+            'search to make sure after I login the user found in the active tabele '
+                .format(user_random_name))
+        self.header_page.search(value=user_random_name)
```

Comment:
not the scope of the test case

Reply:
@0xIslamTaha I want to search to make sure that the record found in the active table after I login

Reply:
I'm sure we discussed this ticket and I asked u to move it to another class. In this class setUp, create a new user, log wit it, and do whatever u wanna